cdflow_commands/plugins/ecs.py

```diff
@@ -61,6 +67,15 @@ def _image_name(self):
             self._release.version or 'dev'
         )
 
+    @property
+    def _latest_image_name(self):
+        return '{}.dkr.ecr.{}.amazonaws.com/{}:{}'.format(
+            self._account_scheme.release_account.id,
+            self._account_scheme.default_region,
+            self._release.component_name,
+            'latest'
+        )
```

Comment:
This is very similar to [lines 61-68](https://github.com/mergermarket/cdflow-commands/blob/f83a2aebf921035e0a181e0f40af46bd2d1b3008/cdflow_commands/plugins/ecs.py#L61-L68); perhaps it could be refactored into a method which takes the tag?

Reply:
I thought about this, but I just thought it was so simple I didn't want to add another method in. Plus I think `_latest_image_name` is a relevant property on the release object, just like `_image_name`